README.md

```diff
@@ -128,7 +128,7 @@ the scripts are executable.
     # when using Oh My Zsh
     vi ~/.zshrc
     # Add the path to dans-dev-tools/ to PATH. You do this by adding a line like this:
-    # export PATH=$PATH:/User/<your-user-name-here>/git/shared/dans-dev-tools
+    # export PATH=$PATH:/Users/<your-user-name-here>/git/service/dans-dev-tools
```

Comment:
@pfrolke :
User -> Users is correct but the other change is not. This is not one of the EASY modules.

Reply:
@janvanmansum I have `dans-dev-tools` under `~/git/service/` as well.
Besides, the '_EASY modules_' are under `~/git/service/easy/`.

I guess the solution proposed by @pfrolke is correct.

Reply:
* `dans-dev-tools` is intended as a generic tool, not just for EASY. It can be used in the new NARCIS modules for example.
* It is not part of the EASY build. `./code-update.py` will not grab it from GitHub.

You can put it where you like: I am just pointing out how I intended it to be used.